### PR TITLE
CHORE: Update build process

### PR DESCRIPTION
**Pull request by roachmd**
#361 

Signed: mroach@3blades.io

### CODE REVIEW

File: config/buildDockerImage.sh [hunk 1]
```diff
@@ -1,14 +1,23 @@
 #!/bin/bash
 
-if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
+if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+  if [ "$TRAVIS_BRANCH" == "master" ]; then
    docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
    docker build --no-cache -t $DOCKER_IMAGE_NAME .;
    docker tag $DOCKER_IMAGE_NAME $DOCKER_IMAGE_NAME:build-$TRAVIS_BUILD_NUMBER;
    docker tag $DOCKER_IMAGE_NAME $DOCKER_IMAGE_NAME:latest;
```
Comment (raygesualdo) [inline config/buildDockerImage.sh]: A thought: what I've commonly seen in the Node ecosystem is that `latest` represents the latest production build, not the most recent dev build. Typically, the most recent dev build is tagged as `next`, `dev`, `beta` or similar. Don't know if this is true in the Docker/Python/elsewhere world, but wanted to mention it.

Comment (johngriebel) [reply config/buildDockerImage.sh]: To me at least, it seems unintuitive to have `latest` refer to anything aside from the literal most recent build image.

Comment (raygesualdo) [reply config/buildDockerImage.sh]: Fair enough.

Comment (raygesualdo) [reply config/buildDockerImage.sh]: I think the driver behind that in the Node space came from using `npm`. When you want the most recent version of a package from the registry, you run `npm install <package>@latest`. The convention stuck.

**Pull request merged**